### ISSUE TITLE
time cost for training?

### ISSUE THREAD

**Issue by charleslipku** [tornadomeet/mxnet-face#19]
I am using the CelebA as training set, which is same as your example. Could you tell me how long it takes to finish the training process?

**Comment by csgcmai** [comment]
Hi there, it depends on your GPU as well as the CPU. The results reported here is trained after 82 epochs for lightened_moon.

**Comment by charleslipku** [comment]
@csgcmai I am wondering if it is possible that I only use single CPU to train the model.

**Comment by csgcmai** [comment]
Well, everything is possible. However, in my case, the speed of GPU (Titan X) is hundreds faster than the CPU (E5-2680). For your reference only.

**Comment by charleslipku** [comment]
@csgcmai Thanks. Then how long it cost for you using the GPU?

**Comment by csgcmai** [comment]
I haven't run the training on these model. You may start a running and check the speed, i.e., images per second. To my experience, it is quite impossible to train the model using the CPU only, while it is fine for the testing.

**Comment by charleslipku** [comment]
@csgcmai Thanks for your information.

**Comment by tornadomeet** [comment]
less than one day with 4 gpu, i forget the accuracy time.
